src/user/room-invitation.ts

```diff
@@ -241,4 +255,19 @@ export class RoomInvitation extends Accessory implements Acceptable {
     return ageSeconds
   }
 
+  public async to (): Promise<Contact> {
+    log.verbose('RoomInvitation', 'to()')
+
+    const payload = await this.puppet.roomInvitationPayload(this.id)
+    const to = this.wechaty.Contact.load(payload.toId)
+    return to
+  }
+
+  public async avatar (): Promise<FileBox> {
+    const payload = await this.puppet.roomInvitationPayload(this.id)
+    const fileBox = FileBox.fromUrl(payload.avatar, `${payload.roomTopic}_avatar.jpg`)
+    await fileBox.ready()
```

Comment:
What's the reason we need a `fileBox.ready()` at here?

Reply:
Because this is a `FileBox` from url, the content is not loaded if no `ready` is called.

Perviously I've met some error thrown when calling some function on `FileBox`, so I get used to adding `ready` function before use the `FileBox`.